src/index.html

```diff
@@ -9,16 +9,16 @@
     <h1>Leaderboard</h1>
     <div id="container">
       <div id="one">
-        <h2>Recent Stories &emsp; <button type="button">Refresh</button></h2>
-        <div class="score_list"> 
-            &emsp;
-        </div>
+        <h2>Recent Stories &emsp; <button class="refresh" type="button">Refresh</button></h2>
+        <ul class="score_list"> 
+          
+        </ul>
       </div>
       <div id="two">
         <h2>Add your score</h2>
-        <input placeholder="Your name" type="text"><br>
-        <input placeholder="Your score" type="text"><br>
-        <button type="button">Submit</button>
+        <input class="user" placeholder="Your name" type="text"><br>
+        <input class="score" placeholder="Your score" type="text"><br>
```

Comment:
- [OPTIONAL] What happens if we dump the input name and score after a user clicks the submit button? This will improve your code 👍🏻